test/e2e/resources/crd.yaml

```diff
@@ -392,6 +392,128 @@ spec:
                       to the Pravega processes as JAVA_OPTS. See the following file
                       for a complete list of options: https://github.com/pravega/pravega/blob/master/config/config.properties'
                     type: object
+                  securityContext:
```

Comment:
Is this from https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#securitycontext-v1-core?